### DISCUSSION TITLE
How do I submit a contribution?

### DISCUSSION THREAD

**Discussion by Meerah-al** [oppia/oppia-android#4680] [Q&A (Other)] [answered]
Good day, I need help. i have completed reviews on my lessons and put them in a file, but how do i submit it?

**Comment by seanlip** [comment, marked as answer]
Hi @Meerah-al, I think you have the wrong discussion board for lesson reviews. Please use [this one](https://github.com/oppia/lessons-team/discussions). Thanks!

**Reply by Meerah-al** [reply]
I am sorry, Thanks
